runtime/sbf/isa.md

```diff
@@ -0,0 +1,264 @@
+# Sealevel Bytecode Format
+
+Sealevel Bytecode Format (SBF) derives from the eBPF little-endian architecture.
```

Comment:
Is it "Sealevel Bytecode Format" or "Solana Bytecode Format"??

Reply:
"Solana Bytecode Format" see https://github.com/solana-labs/solana/issues/20323